glue/stack_trace_base.cc

```diff
@@ -4,12 +4,8 @@
 
 #include "flutter/glue/stack_trace.h"
 
-#include "base/debug/stack_trace.h"
-
 namespace glue {
 
-void PrintStackTrace() {
-  base::debug::StackTrace().Print();
-}
+void PrintStackTrace() {}
```

Comment:
The only place this was used was the CRASH macro in `wtf/Assertions.h`. We rarely aborted because of this. And in case we ever did, the subsequent call tried to dereference 0xfbadbeef which would have given us a backtrace from crash reports anyway. I don't believe we are losing anything valuable here. But if you think we are, I can implement the same again.